documentation/terraform.md

```diff
@@ -61,55 +63,9 @@ getting destroyed. To clean up manually, use the `terraform-app-destroy` make ta
 `pr_id` variable to the PR ID from Github:
 
 ```
-make passcode=MyPasscode pr_id=1234 review terraform-app-destroy
-```
-
-## `terraform plan` with Terraform CLI commands
-
-The equivalent of the Makefile `dev terraform-app-plan` is:
-```
-cd terraform/app
-rm -f .terraform.lock.hcl
-terraform init -reconfigure -backend-config="key=dev/app.tfstate"
-terraform plan -var="paas_sso_passcode=MyPasscode" -var="app_docker_image=dfedigital/teaching-vacancies:dev-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714" -var-file ../workspace-variables/dev.tfvars.json
-```
-
-## Planning out to a file, and using `terraform show`
```

Comment:
This section is interesting to help debugging terraform plan and should be kept (but remove var="paas_sso_passcode=MyPasscode")